src/features/amUI/users/UserItem/UserItem.module.css

```
@@ -62,3 +62,8 @@
 .expandedContent {
   background-color: var(--fds-semantic-surface-neutral-subtle);
 }
+
+.clickableListitem {
```

```suggestion
.clickable {
```
I suggest also renaming the expandable class in line 22 and 26 to `clickable` and then use that class on both the expandable items and the links. That way we get the nice hover and active colors too 😊

---

👍